src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,47 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.enums.Message;
+import duke.exception.DukeException;
+import duke.ui.Ui;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Adds Tasks to the TaskList and invokes appropriate UI messages about it
+ */
+public class AddCommand implements Command {
+    
+    private final String[] parsedInput;
+    private final String commandTag; // indicates what kind of Task
+    
+    public AddCommand(String[] parsedInput) {
+        this.parsedInput = parsedInput;
+        this.commandTag = parsedInput[0];
+    }
+    
+    /**
+     * Prints out a confirmation message of the command, adds the entry to TaskList and displays the current status
+     * of the TaskList
+     * @param tasks Current TaskList
+     * @param ui Where the User shall receive messages about the command
+     * @throws DukeException
```

Comment:
It would be better if you would indicate in the Javadocs the circumstances in which the DukeException would be thrown.